lightning-tag/detailed-reference.md

```diff
@@ -1,3 +1,111 @@
+`getProfileId()` | String | n/a | The id of the profile associated with the browser
+`getAudienceString( [delimiter=','], [limit])` | String | 1. `delimiter` - optional, string, defaults to comma. <br/> 2. `limit` - optional, integer, defaults to no limit | Delimited string of targeting codes or audience ids, based on your established tag settings. The default, i.e. no arguments, is comma delimited with no limit. Example: `"aud1,aud2"` or `"1,2"`
+`getAudiences([limit])` | Array | `limit` - optional, integer, defaults to no limit | Array of either targeting codes as strings or audience ids as integers, based on your established tag settings. Example: `["aud1","aud2"]` or `[123,789]`
+
+### Local Storage
+
+As noted above in the `config` object created at script setup, there is parameter named `audienceLocalStorage`. If passed, when targeting is completed, the audiences will be written out to the browser's localStorage.
+
+Value | Result
+:---: | ------
+`false` | This is the default and will not write audiences to browser localStorage.
+`true` |  Lightning Tag will write audiences to localStorage under the key `lotame_CLIENT_ID_auds` (where CLIENT_ID is your Lotame clientId)
+`'string'` | Passing a string will tell Lightning Tag to write the audiences to localStorage under the key of the string you pass. (i.e. `audienceLocalStorage: 'myLotameAudiences'`)
+
+Audiences are written out as a string of comma-delimited targeting codes or audience ids, based on your established tag settings. Example: `"aud1,aud2"` or `"123,789"`
+
+### getAudiences()
+
+?> Per Mark "which is a helper to retrieve audiences out of local storage". Need more details as I also see this method on the profile object returned by the callback.
+
+## Other Methods
+
+### Collect()
+
+The `collect()` method is available to pass data to your Lotame DMP after the page load is complete and the initial calls to Lotame DMP are finished. An example use-case is tracking user interaction with a video player on your site.
+
+```javascript
+window.lotame_<lotameClientId>.collect({data});
+```
+
+The `data` object parameters are described in the [Data Collection page](lightning-tag/data-collection.md).
+
+Calls to this method are queued up. Every 1 second (and also at the `page.unload()` event) the Lightning Tag flushes the queue and sends the batch of data objects to the DMP. This allows your website to make calls to this method and not need to wait on a response.
+
+### Page()
+
+The `page()` method is another initilization of targeting in your Lotame DMP. This allows for you to re-run your targeting call without reloading of the Lotame Lightning Tag script. A use-case is a single page site where as a user scrolls, new articles pop up. This method allows your site to pass in new `{data}` elements that describe this new article the user is now viewing and then retrieve updated audiences based on that new information.
+
+```javascript
+window.lotame_<lotameClientId>.page({data});
```

Comment:
If you change it to [data], the brackets imply that it is optional

Reply:
Doesn't the brackets imply it is an array though also?